schemas/com.vimeo/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo video.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "video",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "videoId": {
+      "type": "number",
+      "description": "The Vimeo ID of the video.",
+      "minimum": 0,
+      "maximum": 9223372036854776000
+    },
+    "videoTitle": {
+      "type": "string",
+      "description": "The title of the video.",
+      "maxLength": 4096
+    },
+    "videoUrl": {
+      "type": ["string", "null"],
+      "format": "uri",
+      "description": "The URL of the video on vimeo.com.",
+      "maxLength": 4096
+    },
+    "videoWidth": {
```

Comment:
Some of these properties would be really nice to add to the `media_player` schema (or `media_player_quality` for the quality one) to have a common representation across all players. Otherwise we will have a different place for them for each player and there are so many players out there..

Reply:
Good point, it feels like almost all of these properties could/should be moved to a more generic schema, aside from error (but we could also introduce some kind of generic media-error schema?).

Reply:
`videoWidth` and `videoHeight` are included in `meta` schema now.